webapp/channels/src/components/widgets/inputs/check/check.scss

```diff
@@ -7,7 +7,12 @@
     margin-top: 24px;
     margin-bottom: 32px;
 
+    label {
+        cursor: pointer;
```

Comment:
Note that this is non-standard behaviour. I'm not sure why that's the case though, so 0/5, but I thought I'd mention it

Reply:
The affordance helps IMHO. Also not sure why it's not standard for checkbox/radio labels though.